src/components/OrderCreateForm/OrderCreateForm.tsx

```diff
@@ -3,25 +3,16 @@ import { Button } from 'components/Button/Button';
 import { Error } from 'components/Error/Error';
 import { LoaderOverlay } from 'components/LoaderOverlay/LoaderOverlay';
 import { useOrder } from 'hooks/useOrder';
-import { ChangeEvent, FormEvent, ReactElement, useState } from 'react';
+import { FormEvent, ReactElement, useState } from 'react';
 
 export const OrderCreateForm = (): ReactElement => {
-  const [address, setAddress] = useState('');
   const [validationMessage, setValidationMessage] = useState('');
   const { createOrder } = useOrder();
 
   const handleSubmit = (event: FormEvent<HTMLFormElement>): void => {
     event.preventDefault();
-    if (address) {
-      setValidationMessage('');
-      createOrder.mutate({ address });
-    } else {
-      setValidationMessage('Address is required to create an order.');
-    }
-  };
-
-  const handleChange = (event: ChangeEvent<HTMLTextAreaElement>): void => {
-    setAddress(event.target.value);
+    setValidationMessage('');
+    createOrder.mutate();
```

Comment:
Create a validation message if the order isn't created - unable to create order or something